.github/workflows/config.yml

```diff
@@ -33,6 +33,16 @@ jobs:
           git config --global --add safe.directory $(realpath .)
           git diff --exit-code --ignore-cr-at-eol *.xml
 
+      - name: Check measures
+        shell: bash
+        run: |
+          bundle exec rake test_documentation
+          out=$(openstudio tasks.rb update_measures 2>&1)
+          if [[ "$out" == *"RuboCop failed"* ]]; then echo "$out" && exit 1; fi
+          echo "Checking for file differences..."
+          git config --global --add safe.directory $(realpath .)
+          git diff --exit-code --ignore-cr-at-eol *.rb *.xml
```

Comment:
I _think_ I'm okay with this. It will obviously require everyone to run the `update_measures` task every time they push their commits, otherwise there will be CI failures. I suppose we can try it and if we encounter situations where this proves to be a pain, we can comment it out.

I'm thinking this step should be above the "Check HPXMLs" step, no? Check the ruby code is valid first, then update/generate the HPXMLs (using valid ruby).